app/navigation/navigation-core/src/main/kotlin/com/hedvig/android/navigation/core/HedvigDeepLinkContainer.kt

```diff
@@ -18,6 +18,8 @@ interface HedvigDeepLinkContainer {
   val contractWithoutContractId: String
   val contract: String // A specific contract destination with a contractId. If none match, an empty screen is shown
 
+  val terminateInsurance: String
```

Comment:
Try to add a little description here just like the other deep links. Just to make this file easy to read out of context, if you are just looking at the deep links and not where they're used.